pyfstat/core.py

```diff
@@ -371,6 +133,62 @@ def get_output_file_header(self):
         header += self.pprint_init_params_dict()
         return header
 
+    def read_par(
+        self, filename=None, label=None, outdir=None, suffix="par", raise_error=True
+    ):
+        params_dict = helper_functions.read_par(
+            filename=filename,
+            label=label or getattr(self, "label", None),
+            outdir=outdir or getattr(self, "outdir", None),
+            suffix=suffix,
+            raise_error=raise_error,
+        )
+        return params_dict
+
+    def translate_keys_to_lal(self, dictionary):
```

Comment:
Should we really bury these "general" functions into a class?
I can take  "Everything relevant in PyFstat is done through `BaseSearchClass`" as a satisfactory answer.

Reply:
For this one specifically I didn't really see any use case where someone would need to call it manually, as it's just there to resolve some internal variable name mismatch.

For `read_par()` on the other hand I made both an individually-accesible helper function and the class method which just wraps it, because I can easily imagine using it from an external script to read in pulsar ephemerides and then do something nonstandard with the ranges before calling a Writer or XYZSearch class.